lib/src/Uncertainty/Algorithm/MetaModel/openturns/MetaModelValidation.hxx

```diff
@@ -49,19 +49,29 @@ public:
                       const Sample & outputSample,
                       const Function & metaModel);
 
+  /** Constructor with parameters */
+  MetaModelValidation(const Sample & outputSample,
+                      const Sample & metamodelPredictions);
+
```

Comment:
could you mark the previous ctor with "@deprecated" in the comment

Reply:
Ok: fixed at 5700cd3e7ee1dc5a96f9b68a47f87e66625dade9.